cmd/agent/subcommands/run/command.go

```diff
@@ -20,6 +19,8 @@ import (
 	"syscall"
 	"time"
 
+	"github.com/DataDog/datadog-agent/comp/core/workloadmeta/defaults"
```

Comment:
Could you move this line lower so it lives with the rest of the "github.com/DataDog" imports.